src/client/components/features/OffreDeStage/Consulter/ConsulterOffreDeStage.test.tsx

```diff
@@ -36,29 +37,115 @@ describe('ConsulterOffreDeStage', () => {
 
 			expect(intituléOffreDeStage).toBeVisible();
 		});
 
-		it('concernant l‘entreprise du stage', () => {
-			render(<ConsulterOffreDeStage offreDeStage={offreDeStage}/>);
+		it('affiche le nom de l‘employeur', () => {
+			render(<ConsulterOffreDeStage offreDeStage={anOffreDeStage({ employeur: { nom: 'Je suis le nom de l‘employeur' } })}/>);
 
-			const nomEntreprise = screen.getByText('Gras Fisme');
+			const nomEntreprise = screen.getByText('Je suis le nom de l‘employeur');
 
 			expect(nomEntreprise).toBeVisible();
 		});
 
+		describe('description du poste', () => {
+			it('quand elle est fournise, affiche la description du poste', () => {
+				const { getByDescriptionTerm } = render(<ConsulterOffreDeStage offreDeStage={anOffreDeStage({
+					description: 'Je suis une description du poste',
+				})}/>, { queries });
+
+				const descriptionPoste = getByDescriptionTerm('Description du poste :');
+
+				expect(descriptionPoste).toBeVisible();
+				expect(descriptionPoste).toHaveTextContent('Je suis une description du poste');
+			});
+
+			it('quand elle n‘est pas fournise, n‘affiche pas la description du poste', () => {
+				render(<ConsulterOffreDeStage offreDeStage={anOffreDeStage({
+					description: '',
+				})}/>);
+
+				const descriptionPoste = screen.queryByText('Description du poste :');
+
+				expect(descriptionPoste).not.toBeInTheDocument();
+			});
+		});
+		
+		describe('description de l‘employeur', () => {
+			it('lorsqu‘elle est fournise, affiche la description de l‘employeur', () => {
+				const { getByDescriptionTerm } = render(<ConsulterOffreDeStage offreDeStage={anOffreDeStage({
+					employeur: {
+						description: 'Je suis une description de l‘employeur',
+						nom: 'nom',
+					},
+				})}/>, { queries });
+
+				const descriptionEmployeur = getByDescriptionTerm('Description de l‘employeur :');
+
+				expect(descriptionEmployeur).toBeVisible();
+				expect(descriptionEmployeur).toHaveTextContent('Je suis une description de l‘employeur');
+			});
+
+			it('quand elle n‘est pas fournise, n‘affiche pas la description de l‘employeur', () => {
+				render(<ConsulterOffreDeStage offreDeStage={anOffreDeStage({
+					employeur: {
+						description: '',
+						nom: 'nom',
+					},
+				})}/>);
+
+				const descriptionPoste = screen.queryByText('Description de l‘employeur :');
+
+				expect(descriptionPoste).not.toBeInTheDocument();
+			});
+		});
+
+
+		describe('la rémunération du stage', () => {
+			it('lorsque la rémunération n‘est pas renseignée affiche le message attendu', () => {
+				const { getByDescriptionTerm } = render(<ConsulterOffreDeStage
+					offreDeStage={anOffreDeStage({ remunerationBase: undefined })}/>, { queries });
+
+				const remuneration = getByDescriptionTerm('Rémunération :');
+
+
+				expect(remuneration).toBeVisible();
+				expect(remuneration).toHaveTextContent('Non renseigné');
+			});
+			it('lorsqu‘aucune rémunération est proposée affiche le message attendu', () => {
```

Comment:
```suggestion
			it('lorsque la rémunération est à 0, affiche "Aucune"', () => {
```

Juste en lisant le nom du it tel qu'écrit ça ne m'a pas suffit à comprendre la distinction entre les deux tests, c'est pourquoi je propose ce renommage ^^